src/data/abilities/ab-attrs/move-effect-chance-multiplier-ab-attr.ts

```diff
@@ -3,41 +3,36 @@ import { AbAttrFlag } from "#enums/ab-attr-flag";
 import { MoveId } from "#enums/move-id";
 import type { Pokemon } from "#field/pokemon";
 import type { Move } from "#moves/move";
-import type { NumberHolder } from "#utils/common-utils";
+import type { ValueHolder } from "#utils/common-utils";
 
 /**
  * Modifies moves additional effects with multipliers, ie. Sheer Force, Serene Grace.
  * @see {@linkcode apply}
  */
 export class MoveEffectChanceMultiplierAbAttr extends AbAttr {
   private readonly chanceMultiplier: number;
+  /**
+   * Secondary effect chance multipliers do not apply to these moves
+   * even though they are implemented with 100 base chance.
+   */
+  private static exceptMoves = [MoveId.ORDER_UP, MoveId.ELECTRO_SHOT];
```

Comment:
Instead of a `private` `static` class member, you could just make it a non-exported `const` outside of the class. It would minify better that way too.
```suggestion
  private static readonly exceptMoves: readonly MoveId[] = [MoveId.ORDER_UP, MoveId.ELECTRO_SHOT];
```